project/profiles/capacity4more/behat/features/group_dashboard.feature

```diff
@@ -135,3 +135,10 @@ Feature: Group dashboard
     | galileo     |
     | badhairday  |
     | president   |
+
+  @api
+  Scenario: Highlighted document content title should contain a link.
+    Given I am logged in as user "badhairday"
+    When  I visit the dashboard of group "Nobel Prize"
+    Then  I should see the text "Nobel Prize"
+    And I should see the link "Nobel Prize sheet"
```

Comment:
> Not true , it is also appear on the activity stream.

On my installation (from 2 days ago), it does not appear on activity stream, unless you click the 'show more' button:
![anim](https://cloud.githubusercontent.com/assets/17882245/23760390/36dae6a4-04f8-11e7-8fcd-9cc1a92d6fa8.gif)